deploy/terraform/resources.tf

```diff
@@ -0,0 +1,120 @@
+//add or delete items in this list to change what actions will cause notifications
+variable "cloudwatch_filter_events" {
+  type    = "list"
+  default = [
+    "($.eventName!= Test*) &&",
+    "($.eventName!= Reboot*) &&",
+    "($.eventName!= Decrypt*) &&",
+    "($.eventName!= Renew*) &&",
+    "($.eventName!= Switch*) &&",
+    "($.eventName!= Describe*) &&",
+    "($.eventName!= Head*) &&",
+    "($.eventName!= List*) &&",
+    "($.eventName!= Get*) &&",
+    "($.eventName!= Lookup*) &&",
+    "($.eventName!= Estimate*) &&",
+    "($.eventName!= BatchGet*) &&",
+    "($.eventName!= Stop*) &&",
+    "($.eventName!= ExitRole) &&",
+    "($.eventName!= AdminList*) &&",
+    "($.eventName!= AdminGet*) &&",
+    "($.eventName!= AddCommunication*)"
+  ]
+}
+
+
+resource "aws_iam_policy" "cloudwatch_for_lambda" {
+  name = "CloudwatchFilterForLambdaPolicy"
+
+  policy = <<EOF
+{
+  "Version": "2012-10-17",
+  "Statement": [
+    {
+      "Effect": "Allow",
+      "Action": [
+        "iam:ListAccountAliases"
+      ],
+      "Resource": [
+        "*"
+      ]
+    },
+    {
+      "Effect": "Allow",
+      "Action": [
+        "logs:CreateLogGroup",
+        "logs:CreateLogStream",
+        "logs:PutLogEvents"
+      ],
+      "Resource": "arn:aws:logs:*:*:*"
+    }
+  ]
+}
+EOF
+}
+
+
+resource "aws_iam_role" "manual_change_lambda_role" {
+  name = "ManualChangeDetectorLambdaRole"
+
+  assume_role_policy = <<EOF
+    {
+      "Version": "2012-10-17",
+      "Statement": [
+        {
+          "Action": "sts:AssumeRole",
+          "Principal": {
+            "Service": "lambda.amazonaws.com"
+          },
+          "Effect": "Allow",
+          "Sid": ""
+        }
+      ]
+    }
+EOF
+}
+
+
+resource "aws_iam_role_policy_attachment" "cloudwatch_lambda_role_attach" {
+  role = "${aws_iam_role.manual_change_lambda_role.name}"
+  policy_arn = "${aws_iam_policy.cloudwatch_for_lambda.arn}"
+}
+
+
+resource "aws_lambda_function" "manual_change_detector_lambda" {
+  filename = "./terraform/dist/lambdas/manual-change-detector.js.zip"
```

Comment:
I see there is nothing in the project that will generate this file. Is that coming in a later PR?

Reply:
I'm using `lambda-tools build` in our internal version to webpack and create the lambda from that, but since the `lambda-tools` package isn't open sourced I ripped out all deploy pieces for now (besides the tf file). 

Thanks for comments/suggestions btw.

Reply:
💥 https://github.com/lifeomic/lambda-tools 💥

That said, I understand this may mostly be a demonstration to other folks 😄